drivers/adxl372/adxl372.c

```diff
@@ -58,7 +58,7 @@
  * @param reg_data - The register data.
  * @return 0 in case of success, negative error code otherwise.
  */
-int32_t adxl372_spi_reg_read(adxl372_dev *dev,
+int32_t adxl372_spi_reg_read(struct adxl372_dev *dev,
```

Comment:
This looks more like a preference here; and this change adds a bit of noise.
Personally, I don't mind whether we use struct/enum prefixes [Linux style] or go with typedefs;

I guess we'd need to come [internally] to an agreement about which coding style to use for types.

That being said, it's fine from me to add this change.

Reply:
I think we should stick with the Linux coding style. In my opinion, the argument of not using typedefs makes sense: https://www.kernel.org/doc/html/v4.10/process/coding-style.html#typedefs